deploy/manifests/quay-operator/3.4.0/quay-operator.clusterserviceversion.yaml

```diff
@@ -143,9 +143,9 @@ spec:
                 - name: RELATED_IMAGE_COMPONENT_CLAIR
                   value: registry-proxy.engineering.redhat.com/rh-osbs/quay-clair:v3.4.0
                 - name: RELATED_IMAGE_COMPONENT_POSTGRES
-                  value: registry.redhat.io/rhel8/postgresql-10:1
+                  value: registry.redhat.io/rhel8/postgresql-10:latest
```

Comment:
@thomasmckay I've implemented reading these environment variables in a a way which will fail if they are not replaced with manifest `@sha256` references. Will this break that?

Reply:
No, these will be replaced w/ sha by build process.